js/wavesexchange.js

```diff
@@ -1352,10 +1352,10 @@ module.exports = class wavesexchange extends Exchange {
             symbol = market['symbol'];
         }
         const amountCurrency = this.safeCurrencyCode (this.safeString (assetPair, 'amountAsset', 'WAVES'));
-        const price = this.parseNumber (this.priceFromPrecision (symbol, priceString));
-        const amount = this.parseNumber (this.currencyFromPrecision (amountCurrency, amountString));
-        const filled = this.parseNumber (this.currencyFromPrecision (amountCurrency, filledString));
-        const average = this.parseNumber (this.priceFromPrecision (symbol, this.safeString (order, 'avgWeighedPrice')));
+        const price = this.priceFromPrecision (symbol, priceString).toString ();
```

Comment:
Not sure if we need `.toString()` here, `priceFromPrecision` already returns a string.

Reply:
Same story on lines below.